src/main/java/cart/auth/BasicAuthorizationExtractor.java

```diff
@@ -0,0 +1,35 @@
+package cart.auth;
+
+import cart.auth.dto.AuthenticationDto;
+import cart.exception.AuthorizationException;
+import org.apache.tomcat.util.codec.binary.Base64;
+
+public class BasicAuthorizationExtractor {
+
+    private static final String BASIC_TYPE = "Basic";
+    private static final String DELIMITER = ":";
+    private static final int CREDENTIALS_LENGTH = 2;
+
+    public static AuthenticationDto extract(final String authorizationHeader) {
+        if (authorizationHeader.startsWith(BASIC_TYPE)) {
```

Comment:
```suggestion
        if (String.valueOf(authorizationHeader).startsWith(BASIC_TYPE)) {
```
Basic가 들어오지 않을 경우, NPE가 발생하고 있어서요. NPE가 발생하지 않도록 처리해주면 어떨까요~?

Reply:
네 생각지 못했는데 말씀해주셔서 감사합니다. 적용했습니다:)